library/src/main/java/com/afollestad/date/data/DateFormatter.kt

```diff
@@ -51,5 +51,5 @@ internal class DateFormatter {
 
   /** F */
   @CheckResult fun weekdayAbbreviation(calendar: Calendar): String =
-    weekdayFormatter.format(calendar.time).first().toString()
+    weekdayFormatter.format(calendar.time).first().toUpperCase().toString()
```

Comment:
`Locale.getDefault()` should be provided as a parameter to `toUpperCase(...)` 🙂

Reply:
Sry, thought this is a default behavior.

Added 👍